_includes/footer.html

```diff
@@ -1,7 +1,49 @@
 <footer class="realfooter d-print-none" id="footer"><div class="container">
 <div class="row mb-5">
 <div class="col-12 col-md-3 col-lg-1-5">
-<svg class="scalingo-logo-inverse" viewbox="0 0 136.35483 35.9" xmlns="http://www.w3.org/2000/svg"><use xlink:href="#scalingo-logo-full-white"></use></svg></div>
+  <svg viewBox="0 0 111 28" style="height: 36px; width: initial;" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Do you want to add a comment what this SVG is, where does it come from? To ease later read of this code